core/src/main/java/org/apache/iceberg/MetadataUpdateParser.java

```diff
@@ -112,6 +115,12 @@ private MetadataUpdateParser() {}
   // SetLocation
   private static final String LOCATION = "location";
 
+  // AddViewVersion
+  private static final String VIEW_VERSION = "view-version";
```

Comment:
Can you update the REST spec as well?

Reply:
I was planning on updating the REST spec as part of https://github.com/apache/iceberg/pull/7913 because it requires a bunch of other View-specific stuff and it seems easier to update everything in one go